### PR TITLE
✨ Added iframe inside text block for event template (#989)

### PR DESCRIPTION
**Pull request by padms**

### CODE REVIEW

File: studio/schemas/documents/event.ts [hunk 1]
```diff
@@ -37,6 +38,11 @@ export default {
   title: 'Event',
   name: 'event',
   i18n,
+  validation: (Rule:Rule) => Rule.custom((fields:any) => {
+    if (fields.iframe!= null ) return "IFrame is deprecated. Please insert iframe inside the content field."
```
Comment (fernandolucchesi) [inline studio/schemas/documents/event.ts:42]: The formatting is wrong. If you have prettier setup it should auto-format on save. Do you have it installed?

**Review by fernandolucchesi — CHANGES_REQUESTED** [review]

File: web/pageComponents/pageTemplates/Event.tsx [hunk 2]
```diff
@@ -198,13 +198,11 @@ export default function Event({ data }: { data: EventSchema }): JSX.Element {
             )}
             {content && (
               <Content>
-                <RichText value={content}></RichText>
+                <EventText value={content}></EventText>
```
[the remark this answers is not in the document]
Comment (padms) [reply web/pageComponents/pageTemplates/Event.tsx:201]: Done

File: studio/schemas/documents/event.ts [hunk 3]
```diff
@@ -106,13 +101,19 @@ export default {
...
       title: 'Content',
       type: 'array',
-      of: [blockContentType,
-       basicIframe],
+      of: [blockContentType, basicIframe],
     },
     {
       title: 'IFrame (Deprecated)',
       name: 'iframe',
       type: 'basicIframe',
```
Comment (fernandolucchesi) [inline studio/schemas/documents/event.ts:109]: can you also add options: collapsible and collapsable here? :)

Comment (padms) [reply studio/schemas/documents/event.ts:109]: Adding collapsible and collapsed true to basicIframe does not work.

**Pull request merged**